microservice/frontend/catalog/debian/data/etc/nginx/sites-available/test-shop.microservice.io

```diff
@@ -1,3 +1,17 @@
+
+upstream _Site_A {
```

Comment:
@clarsonneur this will send an error on Jenkins job, duplicated _Site_A upstream, I changed this on my PR to be _Test_Site_A and _Test_Site_B then Jenkins worked fine